makeingtables/src/Components/Comments.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import DeleteIcon from '@mui/icons-material/Delete';
+import ImportExportIcon from '@mui/icons-material/ImportExport';
+
+export default class Comments extends React.Component {
+    constructor(props) {
+    super(props);
+  
+    // Initializing the state 
+    this.state = {
+        renderType:'comments',
+        items: [],
+        revData : false
+      };
+  }
+
+  
+  render() {
+    const Comments = this.props.items;
+    return (
+      <div> 
```

Comment:
Fragment if no class of css added in div , as div will add node in DOM whereas Fragment doesn't

Reply:
done